.travis.yml

```diff
@@ -65,7 +65,8 @@ env:
   - RUN_ORDER=filesystem
   - CC_TEST_REPORTER_ID=1d1326c4dfaeede878b46588cda440432d1dd6ec605d2c99af7b240ac7db0477
   matrix:
-  - HEADLESS=false STATIC=false
+  - HEADLESS=false STATIC=false jmri.skipschematests=true
```

Comment:
This will not do what you want it to do since (per http://tldp.org/LDP/abs/html/gotchas.html) periods are not valid characters in bash variables (you can see that the process was launched with the variable `skipschematests` at https://travis-ci.org/JMRI/JMRI/jobs/609974835#L324-L331). You need to make this change in `scripts/travis.sh` instead.

Reply:
Thanks!